pkgs/by-name/ty/typstyle/package.nix

```diff
@@ -36,7 +36,7 @@ rustPlatform.buildRustPackage (finalAttrs: {
   };
 
   meta = {
-    changelog = "https://github.com/Enter-tainer/typstyle/blob/v${finalAttrs.version}/CHANGELOG.md";
```

Comment:
Is this really necessary to change?

Reply:
If you override the version then the changelog link should be updated as well.

Reply:
OK I just see that you removed the suffix `v`. OK I understand, it is to avoid having at two places the same pattern. Good for me.

Reply:
Yeah, exactly as you've said.